cfn-guard/src/lib.rs

```diff
@@ -377,6 +377,70 @@ fn apply_rule_operation(
                 None
             }
         }
+        enums::OpCode::SmallerAs => {
```

Comment:
I think we should have 4 total operations (see my other comment):

< Less Than
> Greater Than
<= Less than or Equal To
>= Greater than or Equal to

Reply:
Added the two additional comparisons and fixed the other issues.

Reply:
Excellent.  Will review this week.  Many thanks for the quick turnaround!